src/App.tsx

```diff
@@ -1,73 +1,30 @@
 import React from 'react';
+import { useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Autocomplete } from './Autocomplete';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+
+  const handleSelectedPerson = (person: Person | null) => {
+    setSelectedPerson(person);
+  };
 
   return (
     <div className="container">
       <main className="section is-flex is-flex-direction-column">
         <h1 className="title" data-cy="title">
-          {`${name} (${born} - ${died})`}
+          {selectedPerson
+            ? `${selectedPerson.name} (${selectedPerson.born} - ${selectedPerson.died})`
+            : 'No selected person'}
```

Comment:
The conditional rendering of `selectedPerson` is correct, but ensure that `selectedPerson` is not null before accessing its properties. The current implementation assumes `selectedPerson` is not null when accessing `name`, `born`, and `died`. Consider using optional chaining or a more explicit check to avoid potential runtime errors.